bootstrap/019_query_history_updates.sql

```diff
@@ -49,7 +49,10 @@ BEGIN
         end if;
 
         DROP TABLE IF EXISTS RAW_QH_EVT ;
-        CREATE TABLE RAW_QH_EVT AS SELECT * FROM INTERNAL_REPORTING.QUERY_HISTORY_COMPLETE_AND_DAILY WHERE filterts >= :oldest_running AND end_time >= :newest_completed;
```

Comment:
the warehouse iupdates also has an ctas

Reply:
gosh darn it. I had that thought, but convinced myself (without looking) that we didn't have a staging table for warehouses :(